File: Proyecto/frontend/nbaAnalitics_fr/src/app/models/elemScoreboard.ts
export class ElemScoreboard{
  id:number;
  team_id:number;
  league_id:number;
  season_id:number;
  standingsdate:string;
  conference:string;
  team:string;
  played:number;
  won:number;
  lost:number;
  won_pct:number;
  home_record:string;
  road_record:string;


  constructor(id:number,
    team_id:number,
    league_id:number,
    season_id:number,
    standingsdate:string,
    conference:string,
    team:string,
    played:number,
    won:number,
    lost:number,
    won_pct:number,
    home_record:string,
    road_record:string){
    this.id = id;
    this.team_id = team_id;
    this.league_id = league_id;
    this.season_id = season_id;
    this.standingsdate = standingsdate;
    this.conference = conference;
    this.team = team;
    this.played = played;
    this.won = won;
    this.lost = lost;
    this.won_pct = won_pct;
    this.home_record = home_record;
    this.road_record = road_record;
  }

}

File: Proyecto/backend/demo/src/main/java/com/example/demo/scoreboard/model/ElemScoreboard.java
package com.example.demo.scoreboard.model;

public class ElemScoreboard {
	private int id;
	private int teamId;
	private String leagueId;
	private int seasonId;
	private String standingsdate;
	private String conference;
	private String team;
	private int played;
	private int won;
	private int lost;
	private double wonPct;
	private String homeRecord;
	private String roadRecord;

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public int getTeamId() {
		return teamId;
	}

	public void setTeamId(int teamId) {
		this.teamId = teamId;
	}

	public String getLeagueId() {
		return leagueId;
	}

	public void setLeagueId(String leagueId) {
		this.leagueId = leagueId;
	}

	public int getSeasonId() {
		return seasonId;
	}

	public void setSeasonId(int seasonId) {
		this.seasonId = seasonId;
	}

	public String getStandingsdate() {
		return standingsdate;
	}

	public void setStandingsdate(String standingsdate) {
		this.standingsdate = standingsdate;
	}

	public String getConference() {
		return conference;
	}

	public void setConference(String conference) {
		this.conference = conference;
	}

	public String getTeam() {
		return team;
	}

	public void setTeam(String team) {
		this.team = team;
	}

	public int getPlayed() {
		return played;
	}

	public void setPlayed(int played) {
		this.played = played;
	}

	public int getWon() {
		return won;
	}

	public void setWon(int won) {
		this.won = won;
	}

	public int getLost() {
		return lost;
	}

	public void setLost(int lost) {
		this.lost = lost;
	}

	public double getWonPct() {
		return wonPct;
	}

	public void setWonPct(double wonPct) {
		this.wonPct = wonPct;
	}

	public String getHomeRecord() {
		return homeRecord;
	}

	public void setHomeRecord(String homeRecord) {
		this.homeRecord = homeRecord;
	}

	public String getRoadRecord() {
		return roadRecord;
	}

	public void setRoadRecord(String roadRecord) {
		this.roadRecord = roadRecord;
	}

}
